source/amazon/installation.rst

```diff
@@ -155,78 +160,45 @@ Plugin configuration
+2. Add the following block of configuration to enable the integration, enter the AWS IAM User credentials you created before and the AWS Account ID of the CloudTrail logs to be processed:
 
 .. code-block:: xml
 
+
     <wodle name="aws-cloudtrail">
       <disabled>no</disabled>
-      <bucket>wazuh-cloudtrail</bucket>
-      <access_key>insert_access_key</access_key>
-      <secret_key>insert_secret_key</secret_key>
-      <remove_from_bucket>no</remove_from_bucket>
       <interval>10m</interval>
       <run_on_start>no</run_on_start>
+      <error_not_skip><error_not_skip>
+      <cloudtrail>
+        <bucket>wazuh-cloudtrail</bucket>
+        <access_key>insert_access_key</access_key>
+        <secret_key>insert_secret_key</secret_key>
+        <aws_account_id>insert_account_id</aws_account_id>
+      </cloudtrail>
     </wodle>
 
+To monitor CloudTrail logs for multiple AWS accounts, configure multiple <cloudtrail> options within the aws-cloudtrail wodle
+
 *Check the user manual reference to read more details about each setting:* :doc:`AWS CloudTrail settings <../user-manual/reference/ossec-conf/wodle-cloudtrail>`
 
 .. note::
 
-        Credentials can be loaded from different locations, you can either specify the credentials as they are in the previous block of configuration or load them from other `Boto3 supported locations. <http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials>`_
+        Credentials can be loaded from different locations, you can either specify the credentials as they are in the previous block of configuration, assume an IAM role, or load them from other `Boto3 supported locations. <http://boto3.readthedocs.io/en/latest/guide/configuration.html#configuring-credentials>`_ (including the use of profiles)
 
 3. Restart your Wazuh system to apply the changes:
 
 .. code-block:: console
 
     # /var/ossec/bin/ossec-control restart
 
-Testing the integration
------------------------
-
-After configuring the module successfully you can expect to see the following log messages in your agent log file: ``/var/ossec/logs/ossec.log``
+Considerations for configuration
+--------------------------------
 
-1. Module starting:
-
-.. code-block:: console
-
-    2018/01/12 18:47:09 wazuh-modulesd:aws-cloudtrail: INFO: Module AWS-CloudTrail started
-
-
-2. Scheduled scan:
-
-.. code-block:: console
+- If the S3 bucket contains a long history of CloudTrail logs, and it is not desired to ingest these old logs into Wazuh, it is recommended to configure the only_logs_after option to skip the older logs on initial execution.
+- The aws-cloudtrail wodle only looks for new logs based upon the key for last processed log object, which includes the datetime stamp.  If older logs are loaded into the S3 bucket or the only_logs_after option date is set to a datetime earlier than previous executions of the wodle, the older log files will be ignored and not ingested into Wazuh.
```

Comment:
I think this point should be more explained. Someone who's starting to use Wazuh may not understand this. I'd put an example or I'd explain how Wazuh looks for logs, sorts them, stores them in DB, etc.

Reply:
Makes sense...